### PR TITLE
update per style guide

### PR DESCRIPTION
**Pull request by ned1313**
In reference to issue #2141.

Updating each example for readability and matching style guide.

Adding video clips for each example.

Add a Next Steps section to push reader towards the other matching rules guides.

### CODE REVIEW

**Comment by ned1313** [comment]
All videos added.

**Comment by mitchdraft** [comment]
I made a bucket for this: https://console.aws.amazon.com/s3/buckets/solo-docs/gloo/?region=us-east-2&tab=overview

I'll upload the videos there and let you know when they're there

**Comment by bettyjunod** [comment]
@mitchdraft will @ned1313 have access to the S3 bucket so he can upload future videos directly himself?

**Comment by mitchdraft** [comment]
yep! I just gave Ned an account with access to that bucket. In the future, we may want to have our bot handle uploads - let me know if you have any issues @ned1313 - I have not uploaded any videos to that bucket yet, I'll leave it to you

**Comment by ned1313** [comment]
Links to the videos have been moved to S3.

File: docs/content/gloo_routing/virtual_services/routes/matching_rules/path_matching/_index.md [hunk 1]
```diff
@@ -143,11 +206,17 @@ glooctl add route --name test-exact-2 --path-exact /posts --dest-name json-upstr
...
+<video controls loop>
+  <source src="https://solo-docs.s3.us-east-2.amazonaws.com/gloo/videos/pathmatch_exacttest_2.mp4"  type="video/mp4">
+</video>
+
 ```shell
 curl -v -H "Host: foo" $(glooctl proxy url)/posts
 ```
 
-Now returns results.
+It will now returns results.
```
Comment (mitchdraft) [inline docs/content/gloo_routing/virtual_services/routes/matching_rules/path_matching/_index.md]: nit: 
```suggestion
It will now return results.
```

**Pull request merged**